opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
```

Comment:
Yes, I added the `build-test` that was missing from `xs-opam` and moved the `install` stanza in a more appropriate place